src/utils/query.js

```diff
@@ -467,7 +467,7 @@ export function orderedFromSnap(snap) {
       const obj = isObject(doc.data())
```

Comment:
I realize this code was not introduce here but lines 460-464 & 467-471 are duplicated code.  Why not write a helper function to call on each `DocumentSnapshot`? @prescottprue 

Might've helped avoid this issue to begin with.